controlplane/controlplane.go

```diff
@@ -53,15 +53,19 @@ func Start(ctx context.Context, config Config) error {
 		return errors.WithStack(err)
 	}
 
-	svc, err := New(ctx, dal.New(conn), config.RunnerTimeout, config.DeploymentReservationTimeout, config.ArtefactChunkSize)
+	dal := dal.New(conn)
```

Comment:
For future reference, naming a variable the same as the package can bite you in the arse sometimes. NBD here.